driver/src/main/java/oracle/nosql/driver/http/Client.java

```diff
@@ -546,6 +553,9 @@ public Result execute(Request kvRequest) {
                 headers.add(HttpHeaderNames.HOST, host)
                     .add(REQUEST_ID_HEADER, requestId)
                     .setInt(CONTENT_LENGTH, contentLength);
+                if (sessionCookie != null && sessionCookie.length() > 0) {
+                    headers.add(COOKIE, SESSION_COOKIE_FIELD + sessionCookie);
```

Comment:
why not include the "session=" in the value of the cookie?

Reply:
ok I can do that. will save an alloc/create per request.